beets/autotag/mb.py

```diff
@@ -517,6 +542,7 @@ def album_for_id(releaseid):
     try:
         res = musicbrainzngs.get_release_by_id(albumid,
                                                RELEASE_INCLUDES)
+        log.info(u'Album ' + str(releaseid) + u' fetched from MusicBrainz')
```

Comment:
I don't think we need this log message. (We already have similar logs elsewhere.)